app/views/fact_values/index.html.erb

```diff
@@ -7,16 +7,16 @@ end
 %>
 <% title_actions documentation_button('3.5.5FactsandtheENC') %>
 
-<table class="<%= table_css_classes('ellipsis')%>">
+<table class="<%= table_css_classes('table-fixed') %>">
   <thead>
     <tr>
       <% unless params[:host_id] %>
         <th><%= sort :host, :as => _("Host") %></th>
       <% end %>
-      <th><%= sort :name, :as => _("Name")  %></th>
+      <th class="col-md-3"><%= sort :name, :as => _("Name")  %></th>
       <th><%= sort :value, :as => s_("FactValue|Value") %></th>
       <th><%= _('Reported at') %></th>
```

Comment:
This can also be limited to `col-md-2` or even `col-md-1` giving us more space for the interesting data.